config/deploy.rb

```diff
@@ -0,0 +1,69 @@
+# config valid only for Capistrano 3.1
+lock '3.2.1'
+
+set :application, 'qna'
+set :repo_url, 'git@github.com:vkurennov/qna.git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.